packages/react-native-device-activity/src/index.ts

```
@@ -307,20 +358,25 @@
   );
 }
 
-export function blockApps(
-  familyActivitySelectionStr?: string,
+export function blockAllApps(triggeredBy?: string): void {
+  return ReactNativeDeviceActivityModule?.blockAllApps(triggeredBy);
+}
+
+export function unblockAllApps(triggeredBy?: string): void {
+  // deprecated, should be renamed to unblockAllApps
+  return ReactNativeDeviceActivityModule?.unblockApps(triggeredBy);
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)